policyengine_canada/variables/gov/provinces/qc/tax/income/credits/children_activities/qc_children_activities_credit.py

```diff
@@ -0,0 +1,39 @@
+from policyengine_canada.model_api import *
+
+
+class qc_children_activities_credit(Variable):
+    value_type = float
+    entity = Household
+    label = "Quebec children's activities tax credit"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
+
+    def formula(household, period, parameters):
```

Comment:
instead of formula we use:
adds = ["qc_non_disabled_children_activities_credit", "qc_disabled_children_activities_credit"]